app/services/client.ts

```diff
@@ -85,16 +90,18 @@ const FetchApi = async ({
         });
       } else {
         console.log("---status--", status);
-        if (response.headers.get("content-type") && response.headers.get("content-type")?.match(/application\/json/)) {
+        if (
+          response.headers.get("content-type") &&
+          response.headers.get("content-type")?.match(/application\/json/)
```

Comment:
? should be with each object and its member objects. The first null check is not needed if we are using ?

Reply:
removed first condition